profiles/OCI Object Storage eu-frankfurt-1.cyberduckprofile

```diff
@@ -0,0 +1,36 @@
+	</string>
+	<key>Region</key>
+	<string>eu-frankfurt-1</string>
+	<key>Regions</key>
+	<array>
+		<string>us-phoenix-1</string>
```

Comment:
As we are connecting to a non-global region endpoint it is probably not advisable to allow creating buckets for different regions – I make the assumption that these region specific requests for new buckets with a region different than the connected endpoint will fail?

Reply:
This is a remnant from my previous testing. Do we need Regions key at all? When it is omitted, does Cyberduck pick the region from Region key while creating top level folders (i.e. buckets)?